sdk/python/jobs/finetuning/standalone/chat-completion/chat_completion_with_model_as_service.ipynb

```diff
@@ -0,0 +1,559 @@
+    "%pip install azure-identity\n",
+    "\n",
+    "%pip install mlflow\n",
+    "%pip install azureml-mlflow"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create AzureML Workspace connections"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import MLClient\n",
+    "from azure.identity import (\n",
+    "    DefaultAzureCredential,\n",
+    "    InteractiveBrowserCredential,\n",
+    ")\n",
+    "\n",
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    credential.get_token(\"https://login.windows.net/<tenant-id>\")\n",
+    "except Exception as ex:\n",
+    "    credential = InteractiveBrowserCredential(tenant_id=\"<tenant-id>\")\n",
+    "\n",
+    "try:\n",
+    "    workspace_ml_client = MLClient.from_config(credential=credential)\n",
+    "except:\n",
+    "    workspace_ml_client = MLClient(\n",
+    "        credential,\n",
+    "        subscription_id=\"<subscription-id>\",\n",
+    "        resource_group_name=\"<resource-group-name>\",\n",
+    "        workspace_name=\"<workspace-name/project-name>\",\n",
+    "    )\n",
+    "\n",
+    "# the models, fine tuning pipelines and environments are available in various AzureML system registries,\n",
+    "# Example: Phi family of models are in \"azureml\", Llama family of models are in \"azureml-meta\" registry.\n",
+    "registry_ml_client = MLClient(credential, registry_name=\"azureml-meta\")\n",
+    "\n",
+    "# Get AzureML workspace object.\n",
+    "workspace = workspace_ml_client._workspaces.get(workspace_ml_client.workspace_name)\n",
+    "workspace._workspace_id"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Pick a foundation model to fine tune\n",
+    "\n",
+    "`Phi-3-mini-4k-instruct` is a 3.8B parameters, lightweight, state-of-the-art open model built upon datasets used for Phi-2. The model belongs to the Phi-3 model family, and the Mini version comes in two variants 4K and 128K which is the context length (in tokens) it can support, we need to finetune the model for our specific purpose in order to use it. You can browse these models in the Model Catalog in the AzureML Studio, filtering by the `chat-completion` task. In this example, we use the `Phi-3-mini-4k-instruct` model. If you have opened this notebook for a different model, replace the model name and version accordingly.\n",
+    "\n",
+    "Note the model id property of the model. This will be passed as input to the fine tuning job. This is also available as the `Asset ID` field in model details page in AzureML Studio Model Catalog."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_name = \"Meta-Llama-3.1-8B-Instruct\"  # \"Phi-3-mini-4k-instruct\"\n",
```

Comment:
the intro is talking about using Phi3 so it's better to use it here, and keep Llama3 in the comment